dandi/dandiapi.py

```diff
@@ -1317,6 +1207,20 @@ def get_digest(
         except KeyError:
             raise NotFoundError(f"No {digest_type} digest found in metadata")
 
+    def get_etag(self) -> Digest:
```

Comment:
`get_digest()` has been publicly documented for a while now; renaming it could break users' code.  Aren't you the one who's usually concerned about backwards compatibility here?

Reply:
yes, I am usually the one whining about backwards compatibility and needing for deprecation cycles ;)  But here since API interface is relatively young, and while we are in 0.x, it would be easier to change it now, and make names more consistent, instead of delaying such a change until later and then possibly causing a larger break.

Reply:
Done: 0da8d93